server/src/main/java/org/elasticsearch/cluster/routing/allocation/ShardsAvailabilityHealthIndicatorService.java

```diff
@@ -155,7 +155,7 @@ public HealthIndicatorResult calculate(boolean verbose, HealthInfo healthInfo) {
         DIAGNOSE_SHARDS_ACTION_GUIDE
     );
 
-    public static final String FIX_DELAYED_SHARDS_GUIDE = "http://ela.st/fix-delayed-shard-allocation";
+    public static final String FIX_DELAYED_SHARDS_GUIDE = "https://ela.st/fix-delayed-shard-allocation";
```

Comment:
I know it's not part of this PR, but this URL does not point to anything (either version of it).